islandora-indexing-fcrepo/src/test/resources/MediaVersionAS2Event.jsonld

```diff
@@ -0,0 +1,53 @@
+{
+   "@context":"https:\/\/www.w3.org\/ns\/activitystreams",
+   "actor":{
+      "type":"Person",
+      "id":"urn:uuid:9029a0c0-d845-4ddd-864c-2198d45839da",
+      "url":[
+         {
+            "name":"Canonical",
+            "type":"Link",
+            "href":"http:\/\/localhost:8000\/user\/1",
+            "mediaType":"text\/html",
+            "rel":"canonical"
+         }
+      ]
+   },
+   "object":{
+      "id":"urn:uuid:4f528a92-4be2-4a9d-85e6-c5f4d36b1bce",
+      "url":[
+         {
+            "name":"Canonical",
+            "type":"Link",
+            "href":"http:\/\/localhost:8000\/media\/7",
+            "mediaType":"text\/html",
+            "rel":"canonical"
+         },
+         {
+            "name":"JSON",
+            "type":"Link",
+            "href":"http:\/\/localhost:8000\/media\/7?_format=json",
+            "mediaType":"application\/json",
+            "rel":"alternate"
+         },
+         {
+            "name":"JSONLD",
+            "type":"Link",
+            "href":"http:\/\/localhost:8000\/media\/7?_format=jsonld",
+            "mediaType":"application\/ld+json",
+            "rel":"alternate"
+         }
+      ],
+      "isNewVersion":"true"
```

Comment:
Ah, shoulda caught this sooner, but "isNewVersion" should be down in the "attachment" section under "content" if we want to follow the schema to the letter of the law.  Not the end of the world (thank goodness the AS2 police aren't a thing), but we should probably get in alignment with the ontology.  I suppose this would affect node versioning as well.  It's probably a lot to address here and now, so I'll make a ticket.  There'll have to be work done in the `islandora` module to get this in the right place.

Reply:
ah ok, i was unaware of the schema for this